src/bin/cargo-miri.rs

```diff
@@ -318,7 +348,12 @@ path = "lib.rs"
     let target = get_arg_flag_value("--target");
     let print_sysroot = !ask_user && has_arg_flag("--print-sysroot"); // whether we just print the sysroot path
     let mut command = xargo();
-    command.arg("build").arg("-q");
+    // This may seen somewhat suprising - we are 'building' libstd
+    // by running (the equivalent of) `cargo check`. It turns out
+    // that `cargo check` has exactly the behavior that we want:
+    // it emits crate metadata (including MIR) without running any
+    // codegen.
+    command.arg("check").arg("-q");
```

Comment:
The `check` vs `build` here just affects the empty "fake crate" that we have for xargo, right? This comment here should be attached to the `cargo_mode` line above, not the command invocation down here.